include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -5560,6 +5560,21 @@ struct OrtApi {
    */
   ORT_API2_STATUS(Graph_GetName, _In_ const OrtGraph* graph, _Outptr_ const char** graph_name);
 
+  /** \brief Get the filepath to the ONNX model from which an OrtGraph is constructed.
```

Comment:
> Get the filepath to the ONNX model

should we limit this to ONNX models or do we also want to support ORT format models? and on that note, I'm not sure if Model::model_path_ is getting set correctly for ORT format models.

Reply:
Updated doc to remove explicit mention of "ONNX". Will look into the question about model_path not being set for ORT format models separately.